app/src/main/java/net/bible/android/control/bookmark/BookmarkControl.java

```diff
@@ -194,12 +198,57 @@ public String getBookmarkVerseKey(BookmarkDto bookmark) {
 		return keyText;
 	}
 
-	public String getBookmarkVerseText(BookmarkDto bookmark) {
+	public String getBookmarkBookUsed(BookmarkDto bookmark) {
+		String bookUsed = "";
+		try {
+			bookUsed = bookmark.getBookUsed();
+		} catch (Exception e) {
+			Log.e(TAG, "Error getting book used", e);
+		}
+		return bookUsed;
+	}
+
+	public Book getBookByAbbreviation(CurrentBiblePage currentBible, String bookUsed) {
```

Comment:
I prefer not to change this now, because 
a) bookUsed is what is stored in the database, not an ID 
b) the code for getBookByAbbreviation tries the initials first anyway 
c) I remember when testing I had to test for both because of issues.

Reply:
> c) I remember when testing I had to test for both because of issues.

I would like to know what kind of issues there are if `initials` is used instead.
If I understand correctly, initials should be used to identify modules. Abbreviation is 
just "nickname", i.e. it is not even guaranteed that it is unique for each module.

Reply:
"bookUsed"is what is displayed on the bookmark list for different translations, and should be the one that the user would best understand (but not loo long because of limited space).  Which is why it cannot be the id. 

I cannot find the example now, but is it not that abbrev was used initially and then later initials got added? So there might still be books where the initials are empty, but the abbreviation is available. This is why I check for initials first, and only if it does not match do I check for a matching abbreviation.

Reply:
Ok, I have an example: If you select the BBE translation (Bible in Bible English), then the abbreviation is BBE and the initials: engBBE1964eb.  The initials is just too long and unreadable. BBE is also the title used when selecting a different translation - so it is what the user see all along. 
Other examples with the same issue are:
A=FBV,I=engfbv2018eb
A=GLW,I=engGLW1996eb
A=GNV,I=enggnv1599eb

Bottom-line: I believe we should stay with using the abbreviation.

Reply:
The clincher for me is that the abbreviation rather than the initials is used when you select another translation. This is what is shown in the top left. I have not seen a place yet where the initials are shown to the user, so he will not be familiar with it.